scripts/websitepreprocess.sh

```diff
@@ -65,9 +65,9 @@ EOT
 done
 
 # 3. All the absolute links needs are directly linking github with the given commit.
```

Comment:
typo in comment ): I know it's not yours, but can we reword?

Reply:
Reworded, hopefully, makes sense.